Presentations/03-data-visualization.Rmd

```diff
@@ -1245,4 +1206,60 @@ https://www.coursera.org/learn/exploratory-data-analysis
 ---
 class: inverse, center, middle
 
-# Thank you~ 
\ No newline at end of file
+# Thank you~ 
+
+---
+class: inverse, center, middle    
+name: appendix
+
+# Appendix
+
+<html><div style='float:left'></div><hr color='#D38C28' size=1px width=1100px></html>
+
+---
+class: inverse, center, middle    
+name: ggplotly
+
+# Interactive graphs
+
+<html><div style='float:left'></div><hr color='#D38C28' size=1px width=1100px></html>
+
+---
+
+# Interactive graphs
+
+There are several packages to create interactive or dynamic data vizualizations with R. Here are a few:
+
+* `leaflet` - R integration tp one of the most popular open-source libraries for interactive maps.
+* `highcharter` - cool interactive graphs.
+* `plotly` - interactive graphs with integration to ggplot.
+* `gganimate` - ggplot GIFs.
+* `DT` - Interactive table
+
+These are generally, html widgets that can be incorporated in to an html document and websites.
+
+---
+
+# Interactive graphs
+
+Now we’ll use the ggplotly() function from the plotly package to create an interactive graph!
+
+### Exercise 6: Interactive graphs.
+
+* Load the plotly package
+* Pass that object with the last plot you created to the ggplotly() function
+
+
+---
+
+# Interactive graphs
+
+```{r, message=FALSE, warning=FALSE, out.width = "70%", out.height="70%", out.extra = 'style="display:block; margin:auto;"'}
+# Load package
+library(plotly)
+
+# Use ggplotly to create an interactive plot
+ggplotly(fig) %>% 
+  layout(legend = list(orientation = "h", x = 0.4, y = -0.2))
+
+```
```

Comment:
This was not compiling for me after the changes in #63. Can you check that it works for you?